040-SDK/160-image-transformations.mdx

```diff
@@ -320,6 +320,40 @@ https://us-east-1.storage.xata.sh/transform/contrast=0.5/4u1fh2o6p10blbutjnphcst
 
 <TransformImageExample url="https://us-east-1.storage.xata.sh/4u1fh2o6p10blbutjnphcste94" options={{ contrast: 0.5 }} />
 
+### download
+
+Depending on the file type, some browsers preview the file instead of downloading it. Setting the `download` property will make the browser to always download the file. The value allows you to assign a name, and optionally an extension to the file as it downloads.
```

Comment:
```suggestion
Depending on the file type, some browsers preview the file instead of downloading it. Setting the `download` property will make the browser to always download the file. The value allows you to assign a name, and optionally an extension to the file that will be downloaded.
```